.env.example

```
@@ -0,0 +1,5 @@
+MONGO_URI=
+JWT_SECRET = 
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)